src/components/User.vue

```diff
@@ -0,0 +1,242 @@
+					{{ additionalHint }}
+				</div>
+
+				<p v-if="!emailIsOptional || email.length > 0" class="input">
+					<input id="email"
+						type="email"
+						class="input__field"
+						name="email"
+						:value="email"
+						disabled>
+					<label for="email" class="infield">{{ t('registration', 'Email') }}></label>
+					<img id="email-icon"
+						class="input__icon"
+						:src="emailIconPath"
+						alt="">
+				</p>
+
+				<p v-if="!emailIsLogin" class="input">
+					<input id="loginname"
+						type="text"
+						name="loginname"
+						class="input__field"
+						:value="loginname"
+						:placeholder="t('registration', 'Login name')"
+						required>
+					<label for="loginname" class="infield">{{ t('registration', 'Login name') }}</label>
+					<img id="loginname-icon"
+						class="input__icon"
+						:src="authIconPath"
+						alt="">
+				</p>
+				<input v-else
+					type="hidden"
+					name="loginname"
+					:value="email">
+
+				<p v-if="showFullname" class="input">
+					<input id="fullname"
+						type="text"
+						name="fullname"
+						class="input__field"
+						:value="fullname"
+						:placeholder="t('registration', 'Full name')"
+						:required="enforceFullname">
+					<label for="fullname" class="infield">{{ t('registration', 'Full name') }}</label>
+					<img id="fullname-icon"
+						class="input__icon"
+						:src="userIconPath"
+						alt="">
+				</p>
+				<input v-else
+					type="hidden"
+					name="fullname"
+					value="">
+
+				<p v-if="showPhone" class="groupmiddle input">
+					<input id="phone"
+						type="text"
+						name="phone"
+						class="input__field"
+						:value="phone"
+						:placeholder="t('registration', 'Phone number')"
+						:required="enforcePhone">
+					<label for="phone" class="infield">{{ t('registration', 'Phone number') }}</label>
+					<img id="phone-icon"
+						class="input__icon"
+						:src="phoneIconPath"
+						alt="">
+				</p>
+				<input v-else
+					type="hidden"
+					name="phone"
+					value="">
+
+				<p class="groupbottom input">
+					<input id="password"
+						type="password"
+						class="input__field"
+						name="password"
+						:value="password"
+						:placeholder="t('registration', 'Password')"
+						required>
+					<label for="password" class="infield">{{ t('registration', 'Password') }}</label>
+					<img id="password-icon"
+						class="svg input__icon"
+						:src="passwordIconPath"
+						alt="">
+					<Button class="toggle-password"
+						type="tertiary-no-background"
+						:aria-label="isPasswordHidden ? t('registration', 'Show password') : t('registration', 'Hide password')"
+						@click.stop.prevent="togglePassword">
+						<template #icon>
+							<Eye v-if="isPasswordHidden" :size="20" />
+							<EyeOff v-else :size="20" />
+						</template>
+					</Button>
+				</p>
+				<Button id="submit"
+					native-type="submit"
+					type="primary"
+					:wide="true"
+					:disabled="submitting"
```

Comment:
I pressed enter on the button without entering a account name and then the button stayed disabled even after entering a name and password.

Reply:
Same with clicking, so registration can not be finished at the moment

Reply:
should work now